playground/stories/index.stories.tsx

```diff
@@ -4,14 +4,17 @@ import { Button } from '../test';
 
 export default {
   parameters: {
-    layout: "centered",
+    layout: 'centered',
   },
 };
 
 export const Button_doc_with_playground = () => (
-  <Playground scope={{ Button }} code={`
+  <Playground
+    scope={{ Button }}
+    code={`
 <Button>Button</Button> 
 <Button primary>Primary</Button> 
 <Button danger>Danger</Button> 
-  `} />
-);
\ No newline at end of file
+  `}
+  />
+);
```

Comment:
file wasn't formatted properly, no real change, just prettifying and then I couldn't revert in the UI